### PR TITLE
More SFA Fixes

### PR DESCRIPTION
**Pull request by SophieMint**
Adds power + cryo tubes.

### CODE REVIEW

**Comment by SophieMint** [comment]
!review

**Comment by SophieMint** [comment]
!bugfix

File: html/changelogs/example copy.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: maptweak
```
Comment (alsoandanswer) [inline html/changelogs/example copy.yml:40]:
```suggestion
changes:
```

**Review by alsoandanswer — COMMENTED** [review]
You should rename the changelog as well.

**Comment by SophieMint** [comment]
Something has broken on my end. Going to close and reopen when I've figured out why this piece of shit VSS is so unhappy.

**Pull request closed without merging**